content/applications/websites/website/publish/translate.rst

```diff
@@ -1,39 +1,85 @@
 
-Once your website is created, you have the opportunity to translate it
-in as many different languages as you want.
+To translate your website, you first have to add the required language(s):
 
-You can only translate your website manually, follow the next step.
+#. Go to your website.
+#. Scroll at the bottom of the page to the **language menu**.
+#. Click the language and select :guilabel:`Add a language`.
 
-Now go to your website. On the bottom right corner of the page, click on
-**Add a language**.
+   .. image:: translate/website-add-language.png
+      :alt: Add a language to your website.
 
-.. image:: translate/translate_website01.png
-    :align: center
+#. Click the :guilabel:`Languages` field and select the required language from the drop-down
+   list. Repeat this step for each additional language.
+#. Click the :guilabel:`Add` button.
 
-Choose the language in which you want to translate your website and then
-click on **Load.**
+.. tip::
+   - You can also edit your website's languages from the backend, in the :guilabel:`Settings`. Go to
+     :menuselection:`Website –> Configuration –> Settings`, and add/remove the required
+     languages in the :guilabel:`Languages` field.
 
-.. image:: translate/translate_website02.png
-    :align: center
+   .. _default-language:
 
-You will see that Now, next to English there is also French, which means
-that the page for the translation has been created. You can also see
-that some of the text has been translated automatically.
+   - The website's default language can be defined in the :guilabel:`Default` field in the
+     :guilabel:`Settings`.
 
-.. image:: translate/translate_website03.png
-    :align: center
+Translating the contents
+========================
 
-To translate the content of the website, click on **Translate** (here
-**Traduire** since we want to translate the website in French).
+Once the required languages have been added, you can translate the contents of your website. To do
+so, go to your website, select the required language from the language menu and click the
+:guilabel:`Translate` button in the upper right part to activate the **Translation mode**.
+
+.. image:: translate/translate-button.png
+   :alt: Translate button
+
+As a result:
+
+- Text strings that have already been translated are highlighted in green;
+- Text strings that need to be translated are highlighted in yellow.
+
+.. image:: translate/website-translation-yellow.png
+   :align: center
+   :alt: Text to be translated highlighted in yellow
+
+You can then replace the original text with the translation directly on your website.
+
+.. tip::
+   - Once the required languages have been installed, you can also translate some items (e.g.,
+     the product's name and description) from the backend (e.g., in the product template). To do so,
+     click the language code (e.g., :guilabel:`EN`) next to the text you want to translate (e.g.,
+     the product name) and edit the required translation.
+
```

Comment:
It was not super clear to me where to go, I thought it would be in backend of Website app, but I was only able to find it in Sales. If it's possible in Website, maybe add more details about where, if it's only in the concerned app, then I'd add this to clarify.

Idk how the examples are normally handled, but perhaps it's a good option to use the example markup and add it under the instructions?